Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'foreman-tasks', '>= 0.7.20'
```

Comment:
Regarding Debian package dependencies, yes. It would also mean that -tasks and all its dependencies can't get updated seperately anymore on Debian without bumping the main foreman package..